### ISSUE TITLE
How about chat group like Telegram ?

### ISSUE THREAD

**Issue by Altiano** [djmaze/resticker#33]
I found that a group chat in Telegram or similar can have much more engagement because many question is just simple.
How about create one?

**Comment by zoispag** [comment]
I would suggest Gitter which is directly related to a GitHub repo

**Comment by djmaze** [comment]
Do be honest, I am not a big fan of support chats. It is really hard to follow along discussions in something like Gitter, because it does not have threads. People who are not attending daily but who might be able to answer a question probably will not see it because it is already lost in the chat's history.

Apart from that, I saw OSS project owners being burned out because it takes much time and energy to follow along and try to help everyone out. So, personally I would probably not invest much time in answering questions in a chat.

I rather imagine a more organized form of discussion, like a discourse forum. (I saw Github is currently experimenting with discussions as a beta feature, but that is not available for normal projects as of yet.)

**Comment by kgnfth** [comment]
> Do be honest, I am not a big fan of support chats. It is really hard to follow along discussions in something like Gitter, because it does not have threads. People who are not attending daily but who might be able to answer a question probably will not see it because it is already lost in the chat's history.
> 
> Apart from that, I saw OSS project owners being burned out because it takes much time and energy to follow along and try to help everyone out. So, personally I would probably not invest much time in answering questions in a chat.
> 
> I rather imagine a more organized form of discussion, like a discourse forum. (I saw Github is currently experimenting with discussions as a beta feature, but that is not available for normal projects as of yet.)

How about Discord ?

**Comment by djmaze** [comment]
Well, Discourse is not much better than Gitter, I believe. Having said that, [Gitter is now joining Matrix](https://matrix.org/blog/2020/09/30/welcoming-gitter-to-matrix), which is an interesting development. (I really favor Matrix over any other chat system.) Also, I realize Gitter in the meantime got threading support, so it could be a much more interesting solution nowadays.

Would be interesting to know how many people are really interested in this. Maybe give a like to this post if you do. But as I said above, I would make no efforts to be regularly available in such a chat, so the community would need to help each other.

**Comment by zoispag** [comment]
Personally I believe "chat rooms" tend to be "temporary". You cannot easily retrieve old discussions and they benefit those online at the moment. I think GitHub Discussions is the way to go.

**Comment by djmaze** [comment]
:+1: I could not have stated it any better.

**Comment by zoispag** [comment]
@djmaze Discussion should now be  [available](https://twitter.com/github/status/1336358966834712578)

**Comment by djmaze** [comment]
Awesome! I just enabled it. Feel free to engage :)
